src/sentry/api/endpoints/rule_snooze.py

```diff
@@ -126,3 +124,53 @@ def post(self, request: Request, project, rule_id) -> Response:
             serialize(rule_snooze, request.user, RuleSnoozeSerializer()),
             status=status.HTTP_201_CREATED,
         )
+
+    def delete(self, request: Request, project, rule_id) -> Response:
+        if not features.has("organizations:mute-alerts", project.organization, actor=None):
```

Comment:
Any reason not to pass the actor here? Can just pass request.user

Reply:
copy paste be like